### DISCUSSION TITLE
Updating alpaca-java to use crypto v1beta3

### DISCUSSION THREAD

**Discussion by jessethouin** [Petersoj/alpaca-java#136] [General]
As the title implies, I have updated this project to use Alpaca's new v1beta3 API, which has quite a few changes (some of which I haven't tested yet). Should I simply create a pull request from my fork? You can view it here - https://github.com/jessethouin/alpaca-java/tree/crypto_v1beta3

**Comment by Petersoj** [comment]
Hey @jessethouin! I really appreciate your contributions. Yes, please create a PR and I can review it in the next week or so. I'd encourage rigorous testing before creating the PR though.

**Reply by jessethouin** [reply]
Done, and rigorous testing in my personal project. :)

https://github.com/jessethouin/strategy/blob/master/src/test/java/com/jessethouin/strategy/StrategyApplicationTests.java

**Reply by grodg** [reply]
@Petersoj @jessethouin I am seeing crypto endpoints returning a 404. I cloned the repo to make the changes, but I couldn't pin point where I need to make the url changes.

**Reply by grodg** [reply]
@jessethouin @Petersoj Nevermind, I was using the crypto endpoint incorrectly. You can ignore this.

**Reply by Petersoj** [reply]
Ok sounds good. I'm glad you were able to resolve you issue.

**Comment by grodg** [comment]
Hello, has anyone discovered any issues from the crypto upgrades? I am seeing 404s from several endpoints.
